spec/system/planning_applications/description_change_validation_request_spec.rb

```diff
@@ -99,11 +99,4 @@
     expect(page).to have_text("approved")
     expect(page).to have_text("Applicant / Agent via Api Wizard")
   end
-
-  it "only displays a new validation request option if application is invalid" do
-    planning_application.update!(status: "in_assessment")
-    click_link "Validate application"
-
-    expect(page).not_to have_content("Start new or view existing validation requests")
-  end
```

Comment:
Why did these get removed? Is this no longer true?

Reply:
The new designs now allow planners to view the change requests regardless of the status, so this link *should* be there when an application is in assessment. Instead, we remove the button for a new request on the validation requests index page